build-system/tasks/release/index.js

```diff
@@ -423,7 +430,14 @@ async function release() {
   log('Discovering release', `${green('flavors')}...`);
   const distFlavors = await discoverDistFlavors_();
 
-  log('Compiling all', `${green('flavors')}...`);
+  if (argv.flavor && distFlavors.length == 0) {
+    log('Flavor', cyan(argv.flavor), 'is inactive. Quitting...');
+    return;
+  }
+
+  if (!argv.flavor) {
+    log('Compiling all', `${green('flavors')}...`);
+  }
```

Comment:
Nit suggestion: If we first do this if block, we can else-if the above with the distFlavors check

Reply:
Ack :)